docs/learn/learn-staking.md

```diff
@@ -244,66 +446,65 @@ nominators as measured by amount of stake allocated to that validator will recei
-  [governance](learn-governance.md) proposals.
-- **Controller** This account acts on behalf of the Stash account, signalling decisions about
-  nominating and validating. It sets preferences like payout account and commission. If you are a
-  validator, it also sets your [session keys](learn-keys.md#session-keys). It only needs enough
-  funds to pay transaction fees.
+In order to be paid your staking rewards, someone must claim them for each validator that you
+nominate. Staking rewards are kept available for 84 eras, which is approximately {{ polkadot: 84 days :polkadot }}{{ kusama: 21 days :kusama }}. For more information on why this is so, see the page on [simple payouts](learn-simple-payouts.md).
+
+:::info Payouts
 
-We designed this hierarchy of separate key types so that validator operators and nominators can
-protect themselves much better than in systems with only one key. As a rule, you lose security
-anytime you use one key for multiple roles, or even if you use keys related by derivation. You
-should never use any account key for a "hot" session key in particular.
+Payouts are unclaimed rewards waiting to be paid out to both validators and nominators. If you go to the Staking payouts page on
+[Polkadot-JS](https://polkadot.js.org/apps/#/staking), you will see a list of all validators
+that you have nominated in the past 84 eras and for which you have not yet received a payout. The payout page is visible only to stakers.
 
-Controller and Stash account keys can be either sr25519 or ed25519. For more on how keys are used in
-Polkadot and the cryptography behind it [see here](learn-keys.md).
+:::
 
-## Validators and Nominators
+Each validator as well as their nominators have the option to trigger the payout for all unclaimed eras. Note that this will pay everyone
+who was nominating that validator during those eras. Therefore, you may not
+see anything in this tab, yet still have received a payout if somebody (generally, but not
+necessarily, another nominator or the validator operator) has triggered the payout for that
+validator for that era.
 
-Since validator slots are limited, most of those who wish to stake their DOT and contribute economic
-security to the network will be nominators.
+:::warning Time limit
 
-Validators do most of the heavy lifting: they produce new block candidates in BABE, vote and come to
-consensus in GRANDPA, validate the state transition function of parachains, and possibly some other
-responsibilities regarding data availability and [XCM](learn-cross-consensus.md).
+If nobody claims your staking rewards within 84 eras, then you will not be able to claim them and they will be lost. Additionally, if the validator unbonds all their own stake, any pending payouts will also be lost. Since unbonding takes {{ polkadot: 28 days :polkadot }}{{ kusama: 7 days :kusama }}, nominators should check if they have pending payouts at least this often.
```

Comment:
```suggestion
If nobody claims your staking rewards within 84 eras, you will not be able to claim them and they will be lost. Additionally, if the validator unbonds all of their self stake, any pending payouts will also be lost. Since unbonding takes {{ polkadot: 28 days :polkadot }}{{ kusama: 7 days :kusama }}, nominators should periodically check if they have pending payouts.
```